src/Endpoint.js

```diff
@@ -60,15 +59,11 @@ class Endpoint<P, T> {
       return new Payload(false, url, this.responseOverride, null)
     }
 
-    let response = null
-    try {
-      response = await (this.mapParamsToBody ? this.postFormData(url, params) : fetch(url))
-    } catch (e) {
-      throw new FetchError({ endpointName: this.stateName, innerError: e })
```

Comment:
Why did you remove FetchError here? Is it used anywhere now?

Reply:
If the FetchError is never triggered then the FetchError should be removed probably.

Reply:
A `FetchError` is thrown in `fetchOrPost` if an error occurs.